Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -337,7 +337,8 @@ WebProcessProxy::~WebProcessProxy()
     WebPasteboardProxy::singleton().removeWebProcessProxy(*this);
 
 #if HAVE(DISPLAY_LINK)
-    processPool().displayLinks().stopDisplayLinks(m_displayLinkClient);
+    if (RefPtr processPool = processPoolIfExists())
+        processPool->displayLinks().stopDisplayLinks(m_displayLinkClient);
```

Comment:
I notice `processPoolIfExists()` will log an error if the process pool doesn't exist, but this is an expected condition in the destructor, so an error is probably not appropriate, right? Alternative:

```suggestion
    if (RefPtr processPool = m_processPool)
        processPool->displayLinks().stopDisplayLinks(m_displayLinkClient);
```

Actually, I have no clue when it is appropriate to use `processPoolIfExists()`. Maybe it should just not log an error? I don't know.

Reply:
Yes, the logging here would be unfortunate. I'll fix.